docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -156,33 +141,31 @@ Estimate calls will also be made in the control panel when using flows like the
   Tax estimate requests sent by BigCommerce may not always contain complete data as these requests will be fired at different stages of the shopper checkout. For example, the **Estimate Shipping & Tax** functionality on the cart page does not provide any billing address data, but the tax provider will still be expected to return a valid tax estimate.
 </Callout>
 
-
 ### Responding to tax estimate requests
 
-When responding to tax estimate requests sent by BigCommerce, tax providers are required to include aggregates and breakdowns of sales tax amounts and rates for product item prices, shipping and handling prices. This is because when navigating to the **Settings > Setup > Tax > Tax Settings** page in the control panel, merchants have the ability to specify whether they would like to show taxes in cart, checkout, orders and invoice **As one summarized line item** or **Broken down by tax rate**.
+When responding to tax estimate requests sent by BigCommerce, you are required to include aggregates and breakdowns of sales tax amounts and rates for product item prices, shipping and handling prices. This is because when navigating to the **Settings > Setup > Tax > Tax Settings** page in the control panel, merchants have the ability to specify whether they would like to show taxes in cart, checkout, orders and invoice **As one summarized line item** or **Broken down by tax rate**.
```

Comment:
```suggestion
When you respond to BigCommerce's requests for tax estimates, you must include aggregates and breakdowns of sales tax amounts and rates for product item prices, shipping and handling prices. This is because merchants can specify whether they want to show taxes in cart, checkout, orders and invoice **As one summarized line item** or **Broken down by tax rate**.
```

^ How merchants navigate the control panel seems irrelevant here (and the linter is telling me the sentence is hard to read)

Reply:
Good call, thanks @bc-andreadao!